<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-DI9KG3PP" failures="5" tests="6" name="CRUDTest" time="0.029" errors="0" timestamp="2023-03-06T22:52:41 ICT">
  <testcase classname="org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest" name="testFunctionGetListCharacterDAOWithDataFromDAO" time="0.004"/>
  <testcase classname="org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest" name="testFunctionGetListCharacterDAOWithDataFromDAO" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [2] but found [123]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [123]
at org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest.testFunctionGetListCharacterDAOWithDataFromDAO(CharacterCRUDTest.java:44)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testFunctionGetListCharacterDAOWithDataFromDAO -->
  <testcase classname="org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest" name="testFunctionGetListCharacterDAOWithDataFromDAO" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
at org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest.testFunctionGetListCharacterDAOWithDataFromDAO(CharacterCRUDTest.java:44)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testFunctionGetListCharacterDAOWithDataFromDAO -->
  <testcase classname="org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest" name="testFunctionGetListCharacterDAOWithDataFromDAO" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [4] but found [3]">
      <![CDATA[java.lang.AssertionError: expected [4] but found [3]
at org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest.testFunctionGetListCharacterDAOWithDataFromDAO(CharacterCRUDTest.java:44)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testFunctionGetListCharacterDAOWithDataFromDAO -->
  <testcase classname="org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest" name="testFunctionGetListCharacterDAOWithDataFromDAO" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [5] but found [4]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [4]
at org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest.testFunctionGetListCharacterDAOWithDataFromDAO(CharacterCRUDTest.java:44)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testFunctionGetListCharacterDAOWithDataFromDAO -->
  <testcase classname="org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest" name="testFunctionGetListCharacterDAOWithDataFromDAO" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [6] but found [5]">
      <![CDATA[java.lang.AssertionError: expected [6] but found [5]
at org.teamswt301.dirverdatabasedata.dao.test.CharacterCRUDTest.testFunctionGetListCharacterDAOWithDataFromDAO(CharacterCRUDTest.java:44)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testFunctionGetListCharacterDAOWithDataFromDAO -->
</testsuite> <!-- CRUDTest -->
